lib/protocol/http/body/reader.rb

```diff
@@ -57,9 +57,9 @@ def finish
 				end
 				
 				# Write the body of the response to the given file path.
-				def save(path, mode = ::File::WRONLY|::File::CREAT, *args)
+				def save(path, mode = ::File::WRONLY|::File::CREAT, **args)
```

Comment:
Should this be `*arguments, **options`?

Reply:
I think with the default, positional `mode` argument, it matches the signature of `File.open`

https://ruby-doc.org/core-3.1.2/IO.html#method-c-open

Reply:
```suggestion
				def save(path, mode = ::File::WRONLY|::File::CREAT, **options)
```